specification/playwrighttesting/PlaywrightTesting.AuthManager/main.tsp

```diff
@@ -179,6 +202,17 @@ union EnablementStatus {
   Disabled: "Disabled",
 }
 
+@doc("The os to configure for remote test runs.")
+union OS {
+  string,
+
+  @doc("Linux OS.")
+  Linux: "linux",
+
+  @doc("Windows OS.")
+  Windows: "windows",
```

Comment:
Not a blocker, but we'd expect the enum values be consistent. Here is it lowercase e.g. `linux`, and the enum value above is camelCase e.g. `Active`. Any reason for this inconsistency?

Reply:
Agreed @weidongxu-microsoft but unfortunately this is already being used in the service as is so we thought to add them in these forms. We can probably support consistent form in contract and convert in backend but that would require backend changes. Do you suggest we update here and support in service later?

Reply:
I've updated the values in the contract for now.